x-pack/test/functional/apps/maps/group1/documents_source/search_hits.js

```diff
@@ -120,7 +120,7 @@ export default function ({ getPageObjects, getService }) {
         const { lat, lon, zoom } = await PageObjects.maps.getView();
         expect(Math.round(lat)).to.equal(43);
         expect(Math.round(lon)).to.equal(-102);
-        expect(Math.round(zoom)).to.equal(5);
```

Comment:
With this change the test fails in Firefox head run:

```
1)    maps app
       documents source
         search hits
           layer query
             should apply layer query to fit to bounds:

      Error: expected 5 to equal 4
```

I suggest to put `skipFirefox` tag or add a comment, so that it won't become a surprise for somebody in the future